packages/language-server/server/completion/type-aliases.js

```diff
@@ -0,0 +1,128 @@
+const globalTypeAliases = [
```

Comment:
Can we make it not like hardcode, but get all from `globalScope.typeScope`?

Reply:
I did not see `globalScope` in `ModuleScope` or `PositionedModuleScope`... Also in global `typeScope` (which does not have `parent`) are many type aliases without name. There are many functions signatures.

Reply:
Also type `null` is given as a value, not *string*. Is this right?

Reply:
Yes, it is